.env.example

```diff
@@ -19,6 +19,7 @@ JWT_EXPIRY_TIME="7 days"
 
 # --- BLOCKCHAIN --- #
 MNEMONIC="stone eyebrow bronze glide upon subject camera cushion rice outside swing casual"
+REACT_APP_MNEMONIC="stone eyebrow bronze glide upon subject camera cushion rice outside swing casual"
```

Comment:
Let's store the issuer address directly in the env variable instead of storing the whole mnemonic.